tests/functional/steps/then.py

```diff
@@ -48,11 +47,6 @@ def check_value_in_response_body_at_path(response_body: dict, value: str, path:
             assert match.value == value, f'{match.value} is not the expected value, {value}, at {path}'
 
 
-@then('the response body contains the expected response')
-def response_body_as_expected(response_body: dict, patient: Patient) -> None:
-    assert response_body == patient.expected_response
-
-
```

Comment:
See above comments about checking full body of the response. This step is no longer required.